infra/src/main/kotlin/fi/fta/geoviite/infra/switchLibrary/SwitchLibraryService.kt

```diff
@@ -64,4 +70,39 @@ class SwitchLibraryService(
 
     private fun getOrThrow(id: IntId<SwitchStructure>) =
         structuresById[id] ?: throw NoSuchEntityException(SwitchStructure::class, id)
+
+    fun areSame(s1: SwitchStructure, s2:SwitchStructure):Boolean {
```

Comment:
Formatointi välilyönti `:` jälkeen. Tämä on kyllä tyhmää että näitä joutuu tekemään käsin. Pitäisi varmaan koittaa vielä kerran etsiä kotlinille toimiva formatter.

Reply:
Joo, olisi kyllä ihan hyvä. Sekin auttais, jos se laittais nämä yksinkertaiset asiat kuntoon, vaikkei tekisikään niin mullistavia muutoksia kuin UI puolella.